<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectOptionsWidget</class>
 <widget class="QWidget" name="SelectOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>104</width>
    <height>80</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="title_lbl">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>SELECT OPTIONS</string>
     </property>
     <property name="margin">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="select_move_layout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="schrodinger::sketcher::ModularToolButton" name="select_btn">
       <property name="maximumSize">
        <size>
         <width>36</width>
         <height>34</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton { border : none; }</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>36</width>
         <height>34</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="popupMode">
        <enum>QToolButton::DelayedPopup</enum>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">select_move_group</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="move_rotate_btn">
       <property name="maximumSize">
        <size>
         <width>36</width>
         <height>34</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Rotate and translate structure</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton { border : none; }</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../sketcher.qrc">
         <normaloff>:/icons/select_move_rotate.svg</normaloff>
         <disabledoff>:/icons/select_move_rotate_dis.svg</disabledoff>:/icons/select_move_rotate.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>36</width>
         <height>34</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">select_move_group</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="clear_delete_layout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="select_all_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Select all</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton {
border : none;
}
QToolButton:hover {
color: #5b8aa8;
background-color: transparent;
}</string>
       </property>
       <property name="text">
        <string>All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="clear_selection_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Clear selection</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton {
border : none;
}
QToolButton:hover {
color: #5b8aa8;
background-color: transparent;
}</string>
       </property>
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="delete_selection_btn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Delete selected structure</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton {
border : none;
}
QToolButton:hover {
color: #5b8aa8;
background-color: transparent;
}</string>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>schrodinger::sketcher::ModularToolButton</class>
   <extends>QToolButton</extends>
   <header>schrodinger/sketcher/modular_tool_button.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../sketcher.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="select_move_group"/>
 </buttongroups>
</ui>
